planning/behavior_path_planner/behavior_path_planner_pull_out-design.md

```diff
@@ -0,0 +1,131 @@
+- Combine this path with center line of road lane
+
+![shift_pull_out](./image/shift_pull_out.drawio.svg)
+
+[shift pull out vidoe](https://user-images.githubusercontent.com/39142679/187872468-6d5057ee-e039-499b-afc7-fe0dc8052a6b.mp4)
+
+###### parameters for shift pull out
+
+| Name                            | Unit   | Type   | Description                                                                                                          | Default value |
+| :------------------------------ | :----- | :----- | :------------------------------------------------------------------------------------------------------------------- | :------------ |
+| enable_shift_pull_out           | [-]      | bool   | flag whether to enable shift pull out                                                                                | true          |
+| shift_pull_out_velocity         | [m/s]  | double | velocity of shift pull out                                                                                           | 2.0           |
+| pull_out_sampling_num           | [-]      | int    | Number of samplings in the minimum to maximum range of lateral_jerk                                                  | 4             |
+| maximum_lateral_jerk            | [m/s3] | double | maximum lateral jerk                                                                                                 | 2.0           |
+| minimum_lateral_jerk            | [m/s3] | double | minimum lateral jerk                                                                                                 | 0.5           |
+| minimum_shift_pull_out_distance | [m]    | double | minimum shift pull out distance. if calculated pull out distance is shorter than this, use this for path generation. | 20.0          |
+
+##### **geometric pull out**
+
+Generate two arc paths with discontinuous curvature. Ego-vehicle stops once in the middle of the path to control the steer on the spot.
+See also [[1]](https://www.sciencedirect.com/science/article/pii/S1474667015347431) for details of the algorithm.
+
+![geometric_pull_out](./image/geometric_pull_out.drawio.svg)
+
+[geometric pull out video](https://user-images.githubusercontent.com/39142679/181024707-3e7ca5ee-62de-4334-b9e9-ded313de1ea1.mp4)
+
+###### parameters for geometric pull out
+
+| Name                        | Unit  | Type   | Description                                             | Default value |
+| :-------------------------- | :---- | :----- | :------------------------------------------------------ | :------------ |
+| enable_geometric_pull_out   | [-]     | bool   | flag whether to enable geometric pull out               | true          |
+| divide_pull_out_path   | [-]     | bool   | flag whether to devide arc paths.　The path is assumed to be divided because the curvature is not continuous. But it requires a stop during the departure.                | false          |
```

Comment:
typo

Reply:
fix in https://github.com/autowarefoundation/autoware.universe/pull/2948/commits/339db78f30baa093e2f67a5ef52797cf39c1c2d8